addon/services/segmentio.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+// Should match methods in snippet-core.min.js
+const SEGMENT_METHODS = [
+  'trackSubmit',
+  'trackClick',
+  'trackLink',
+  'trackForm',
+  'pageview',
+  'identify',
+  'reset',
+  'group',
+  'track',
+  'ready',
+  'alias',
+  'page',
+  'once',
+  'off',
+  'on'
+];
+
+export default Ember.Service.extend({
+  init() {
+    if (!window.analytics) { return; }
+    SEGMENT_METHODS.forEach((methodName) => {
+      this[methodName] = window.analytics[methodName];
```

Comment:
Isn't this potentially buggy because you're not binding? You may want to do:

```javascript
this[methodName] = window.analytics[methodName].bind(window.analytics);
```